### PR TITLE
pkg-export-k8s: Add status reporting

### PR DESCRIPTION
**Pull request by zeenix**
Currently the kubernetes exporter doesn't report anything about what its
doing. Let's add some basic status reporting.

### CODE REVIEW

File: components/pkg-export-kubernetes/src/lib.rs [hunk 1]
```diff
@@ -55,13 +55,37 @@ pub fn export_for_cli_matches(ui: &mut UI, matches: &clap::ArgMatches) -> Result
...
+                Status::Created,
+                format!("Kubernetes manifest file {}", o),
+            )?;
+
+            file
+        }
+        _ => {
+            let stdout = Box::new(io::stdout());
+            ui.status(
+                Status::Custom('→', String::from("Writting")),
```
Comment (christophermaier) [inline components/pkg-export-kubernetes/src/lib.rs]: "Writing"

Comment (zeenix) [reply components/pkg-export-kubernetes/src/lib.rs]: Can't ever get this word right. :)

Comment (zeenix) [reply components/pkg-export-kubernetes/src/lib.rs]: Fixed. Since it was a trivial fix,  I forced push.

**Comment by christophermaier** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/356968652).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/356968652).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**